### PR TITLE
Tim/eng 3609 share brc20 validation logic in core

### PR DESCRIPTION
**Pull request by teebszet**
# 🔘 PR Type
- [x] Refactoring (no functional changes, no API changes)

# 📜 Background
move brc20 validation logic into core

# 🔄 Changes
depends on https://github.com/secretkeylabs/xverse-core/pull/358

# 🖼 Screenshot / 📹 Video
Include screenshots or a video demonstrating the changes. This is especially helpful for UI changes.

# ✅ Review checklist
Please ensure the following are true before merging:

- [ ] Code Style is consistent with the project guidelines.
- [ ] Code is readable and well-commented.
- [ ] No unnecessary or debugging code has been added.
- [ ] Security considerations have been taken into account.
- [ ] The change has been manually tested and works as expected.
- [ ] Breaking changes and their impacts have been considered and documented.
- [ ] Code does not introduce new technical debt or issues.

### CODE REVIEW

File: src/app/screens/ordinals/ordinalImage.tsx [hunk 1]
```diff
@@ -237,7 +241,8 @@ function OrdinalImage({
     );
   }
 
-  if (contentType.includes('image')) {
+  // if content type is undefined or "", we treat it the same as an image and fall back to ordiview thumbnail
+  if (!contentType || contentType.includes('image')) {
```
Comment (fedeerbes) [inline src/app/screens/ordinals/ordinalImage.tsx:245]: dunno what PR will be merged first but [this](https://github.com/secretkeylabs/xverse-web-extension/pull/777/commits/89a84308929e7bc72689b1671908c17f3ccd2b02) change maybe overwritten right?

**Review by fedeerbes — APPROVED** [review]
nice 🚀.

File: src/app/screens/ordinals/ordinalImage.tsx (hunk 1 above)
Comment (teebszet) [reply src/app/screens/ordinals/ordinalImage.tsx:245]: yup.. good catch

**Comment by fedeerbes** [comment]
@teebszet will merge this one into #777. so we have everything in one place

**Pull request merged**